AmplifyPlugins/Storage/Sources/AWSS3StoragePlugin/Service/Storage/AWSS3StorageService.swift

```diff
@@ -104,6 +105,8 @@ class AWSS3StorageService: AWSS3StorageServiceBehaviour, StorageServiceProxy {
         self.awsS3 = awsS3
         self.bucket = bucket
 
+        self.userAgent = "\(AmplifyAWSServiceConfiguration.frameworkMetaData())"
```

Comment:
This does not seem to have the same data and format as V1's `AmplifyAWSServiceConfiguration.baseUserAgent()`. 

Is that OK/expected?

v1: `amplify-iOS/1.28.1 iOS/16.0 en_US`
v2: `lib/amplify-ios/1.27.1-swift-sdk-dev-preview.0`

Reply:
Here is the useragent string from Swift SDK:

```aws-sdk-swift/1.0 api/cognito-identity-provider/1.0 os/macOS/12.6.0 lang/swift/5.7 lib/amplify-ios/1.27.1-swift-sdk-dev-preview.0, ```